### ISSUE TITLE
Vertex color being doubled upon saving

### ISSUE THREAD

**Issue by zrksyd** [ScanMountGoat/ssbh_data_py#5]
```
import ssbh_data_py
mesh = ssbh_data_py.mesh_data.read_mesh("model.numshb")
mesh.save("model.numshb")
```
The file used is stage/battlefield_s/normal/model/floating_plate_set/model.numshb

**Before:**
![Before](https://user-images.githubusercontent.com/6856627/119928081-dbbbca80-bf48-11eb-977b-41dfb9538305.png)
**After:**
![After](https://user-images.githubusercontent.com/6856627/119928108-e9715000-bf48-11eb-8663-3780e0d5da36.png)

**Comment by ThatNintendoNerd** [comment]
Aside from the vertex color information, data at 0x1E8008 in the example file also begins to differ from the original file. The mesh names are being stored in the incorrect order and the string pointers' relative addresses are shifted as a result.

**Comment by ScanMountGoat** [comment]
> Aside from the vertex color information, data at 0x1E8008 in the example file also begins to differ from the original file. The mesh names are being stored in the incorrect order and the string pointers' relative addresses are shifted as a result.
The example modifies the attribute data and adds an additional attribute, so it won't be binary identical with the original. ssbh_data will often not be exactly 1:1 due to differences in precision for attribute data, floating point rounding errors, etc. 

I'm generating the rigging data in a slightly different order (per mesh rather than per buffer), which I'll adjust to preserve ordering for existing files. The code is using ssbh_lib for exporting, so I'm not too worried about offsets shifting. It's probably due to the other file changes like the rigging buffers changing order.

**Comment by ScanMountGoat** [comment]
The vertex color doubling is definitely a bug. It's an easy fix, but you'll have to multiply by 0.5 or divide by 2 for now until the next release.
